src/System.Private.CoreLib/shared/System/Globalization/ChineseLunisolarCalendar.cs

```diff
@@ -2,54 +2,29 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-
 namespace System.Globalization
 {
-    /*
-    **  Calendar support range:
-    **      Calendar               Minimum             Maximum
-    **      ==========             ==========          ==========
-    **      Gregorian              1901/02/19          2101/01/28
-    **      ChineseLunisolar       1901/01/01          2100/12/29
-    */
+    /// <summary>
+    /// Calendar support range:
```

Comment:
Nit: This comment is not a good "summary" for the type. "summary" is meant to be what you want to see in Intellisense popup when you hover over the type with mouse cursor.

Reply:
(it can stay as regular comment)

Reply:
I think it makes sense as remarks - updated where appropriate